redash/query_runner/big_query.py

```diff
@@ -242,14 +242,20 @@ def _get_query_result(self, jobs, query):
 
         return data
 
-    def _get_columns_schema(self, table_data):
+    def _get_columns_schema(self, table_data, table_type):
```

Comment:
do this change break other data sources ? 
what about adding a default value ? 

It's only in big query query runner but stating this so you can validate.

Reply:
that object is just thrown into javascript, javascript handles missing fields as nulls(or rather undefined but for our use that's more or less the same). So it's not going to break anything.